lib/src/cli.rs

```diff
@@ -130,7 +131,12 @@ enum ContainerOpts {
 struct ContainerProxyOpts {
     #[structopt(long)]
     /// Path to Docker-formatted authentication file.
-    authfile: Option<String>,
+    authfile: Option<PathBuf>,
+
+    #[structopt(long)]
+    /// Directory with certificates (*.crt, *.cert, *.key) used to connect to registry
+    /// Equivalent to `skopeo --cert-dir`
```

Comment:
Is this too long for the help text?

Reply:
Hmm.  I just tried this out and it looks like structopt eats newlines, but does do auto-wrapping.  I just tried e.g.:

```
diff --git a/lib/src/cli.rs b/lib/src/cli.rs
index e6cbaee..9726fcb 100644
--- a/lib/src/cli.rs
+++ b/lib/src/cli.rs
@@ -162,6 +162,7 @@ enum ContainerImageOpts {
     },
 
     /// Copy a pulled container image from one repo to another.
+    /// Blah blah.  Lorem ipsum foo bar baz. Lorem ipsum foo bar baz. Lorem ipsum foo bar baz. Lorem ipsum foo bar baz.
     Copy {
         /// Path to the source repository
         #[structopt(long)]
```

And I get:

```
walters@toolbox /v/s/w/s/g/o/ostree-rs-ext (main)> cargo b; and ./target/debug/ostree-ext-cli container image --help
   Compiling ostree-ext v0.5.1 (/var/srv/walters/src/github/ostreedev/ostree-rs-ext/lib)
   Compiling ostree-ext-cli v0.1.4 (/var/srv/walters/src/github/ostreedev/ostree-rs-ext/cli)
    Finished dev [optimized + debuginfo] target(s) in 6.94s
ostree-ext-cli-container-image 0.5.1
Commands for working with (possibly layered, non-encapsulated) container images

USAGE:
    ostree-ext-cli container image <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

SUBCOMMANDS:
    copy      Copy a pulled container image from one repo to another. Blah blah.  Lorem ipsum foo bar baz. Lorem
              ipsum foo bar baz. Lorem ipsum foo bar baz. Lorem ipsum foo bar baz
    deploy    Perform initial deployment for a container image
    help      Prints this message or the help of the given subcommand(s)
    list      List container images
    pull      Pull (or update) a container image
walters@toolbox /v/s/w/s/g/o/ostree-rs-ext (main)> 
```

Soo...it should be fine if you just add a period `.` at the end of sentences.  Or it could be:
```
/// Directory with certificates (*.crt, *.cert, *.key) used to connect to registry; equivalent to `skopeo --cert-dir`.
```
or so?

Reply:
Never added the period but not sure that's important